runtime.md

```diff
@@ -27,9 +27,10 @@ This is provided so that consumers can find the container's configuration and ro
 
 ```json
 {
+    "version": "0.2",
```

Comment:
This is handled by #144, which I've just rebased to reflect the current 0.2.0 version.